dotnet/samples/KernelSyntaxExamples/README.md

```diff
@@ -180,3 +180,75 @@ Apim__SubscriptionKey
 Postgres__ConnectionString
 Redis__Configuration
 ```
+
+# Authentication for the OpenAPI Functions
+
+The Semantic Kernel OpenAPI Function enables developers to take any REST API that follows the OpenAPI specification and import it as a plugin to the Semantic Kernel. However, the Kernel needs to be able to authenticate outgoing requests per the requirements of the target API. This document outlines the authentication model for the OpenAPI plugin.
```

Comment:
It looks like the auth provider code described by this README section is now in Example 24. Should we put a reference/link to that example code here?

Reply:
Thanks, that's a good idea. We are reorganizing our examples, and most probably there will be new location of that example very soon. I will update this README as soon as new structure of examples is available. Thank you!